pkg/controllers/scheduler/framework/util.go

```diff
@@ -481,3 +498,21 @@ func DefaultNormalizeScore(maxPriority int64, reverse bool, scores ClusterScoreL
 	}
 	return nil
 }
+
+func GetRelevantResources(su *SchedulingUnit) []corev1.ResourceName {
+	resources := make([]corev1.ResourceName, 0, len(DefaultRequestedRatioResources))
+	for resourceName := range DefaultRequestedRatioResources {
```

Comment:
Most of the resource names follow the the same rules.Do we need these kinds of judgement?

Reply:
This logic is to mainly to keep the logic untouched without GPU requests.
For example, if we only has cpu and memory and no modification is made, it will become `(cpu * 1 + memory * 1 + 0 * 4) / 6`. This would not meet our expectations.